src/dashboard/Dashboard.tsx

```diff
@@ -1,41 +1,112 @@
-import { Card, Paper, CardContent, Container, Box } from "@mui/material";
+import {
+  Card,
+  Paper,
+  CardContent,
+  Container,
+  Box,
+  Divider,
+} from "@mui/material";
+import AttachMoneyIcon from "@mui/icons-material/AttachMoney";
+import ShoppingCartIcon from "@mui/icons-material/ShoppingCart";
+import CommentIcon from "@mui/icons-material/Comment";
+import PersonAddIcon from "@mui/icons-material/PersonAdd";
+import { DashboardCardHeader } from "./DashboardCardHeader";
+import { useGetList, ListContextProvider, Datagrid } from "react-admin";
+import CustomerField from "../customer/CustomerField";
+import { Customer } from "../customer/customer";
+
+const cardSx = {
+  minWidth: "230px",
+};
+
+const getMonthsBeforeNow = function (months: number): Date {
+  const d = new Date();
+  d.setMonth(d.getMonth() - months);
+  return d;
+};
+const oneMonthBeforeNow = getMonthsBeforeNow(1);
+const twoMonthsBeforeNow = getMonthsBeforeNow(2);
 
 export const Dashboard = () => {
+  const monthlyOrders = useGetList("commands", {
+    filter: { date_gte: oneMonthBeforeNow.toISOString() },
+  });
+  const pendingOrders = useGetList("commands", {
+    filter: { status: "ordered" },
+    pagination: { page: 1, perPage: 25 },
+  });
```

Comment:
I doubt we care about pending orders older than a month. Maybe you can merge this request with the previous one?